aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -473,10 +491,16 @@ module AptosFramework::Stake {
     ) acquires StakePool, StakePoolEvents, ValidatorSet, ValidatorSetConfiguration {
         assert!(owner_cap.pool_address == pool_address, Errors::invalid_argument(ENOT_OWNER));
 
-        let stake_pool = borrow_global_mut<StakePool>(pool_address);
+        // Short-circuit if amount added is 0 so we don't emit events.
```

Comment:
this feels more like an abort than silent return

Reply:
Updated